### DISCUSSION TITLE
Assign a task to several people.

### DISCUSSION THREAD

**Discussion by Community-Mailex** [Leantime/leantime#796] [Ideas]
Hi to you champions!

I'm not a regular on the forums but I'm coming here to congratulate you for your work. The software is simply amazing. Thanks to you, we will greatly improve our productivity because free solutions often tend to be quite complex, where your solution is clear and ergonomic.

We think it would be nice to be able to assign a task to several people. This is the only thing that slowed us down a bit. Nothing serious, but it pushes us to multiply the tasks so that all the people are concerned, reducing the possibilities of follow-up of a task. For example, we would like to be able to assign a task to someone but have a person in charge to supervise it.

I wish you a good continuation and do not hesitate to contact me if you want feedback on features.

**Comment by marcelfolaron** [comment]
Thank you for your feedback. 

To be honest we are going back and forth on this particular issue and I'd like to open it up for discussion.
My personal take is that adding multiple people to the same To-Do will eventually lead to a lack of accountability. You could get into a situation where the first person is expecting the 2nd person to do something or vice versa. My philosophy has been to split up the To-Do into the "actual work" that each individual needs to perform. 

With that said, a lot of people are requesting this feature and I can see a use case where multiple people need to perform the same task. But in that case, who would decide if a task is done, change status etc? Is that communicated between the two? 

I don't want to impose a limitation if the community really needs it, I just want to make sure we are actually providing a tool that will ensure ownership and productivity. 

Thank you for your input.

**Comment by Saisnat** [comment]
Hello,

I mean this feature can be used if 2 people or more are working on the same stuff. But the way it should be meant is 1 big task attributed to two let's make an example:

- Task is created and attributed to 2 people.
- The idea now is for those 2 people to create sub-tasks and each one does one sub-task.
- Then they update the status of their sub-task.
- Also since there is a discussion in each major task they can just update each one on the progress.

On my personal side, I would use this feature for that so I can let my members know that one task needs 2 people to do the sub-tasks.

**Comment by datWeazel** [comment]
I understand that assigning a task to several people could lead to situations where two people get assigned the same task and both think that the other person will do the task so nobody does it. But we are currently using leantime for a hobby project and we are coordinating work between around 10 people with it. There are often tasks that we assign to multiple people and then these people freely split the workload between eachother. We don't want to force anyone since it's just a hobby so we have "sub teams" (or whatever you want to call them) and we rather say "Subteam A has to do X" and most of the time it works pretty well for us.

So we would fully support a way to assign a task to multiple people.

**Reply by marcelfolaron** [reply]
Thank you for the additional input. We will revisit the decision. I like the use case of using subtasks to break up large tasks.

**Comment by diderich** [comment]
I think, adding at *Team* or *Group* concept (to which one or more users can be assigned), in addition to *Users* to which tasks, ... could be assigned, could be a solution. It could also help implement and object-based authorization system, i.e., where a *team/group* gets access to canvas, idea boards, ..., rather than all members of a project.

**Comment by joel72265** [comment]
What about using Project Scores (or points) for each task (or even better the Project). The higher the score the user accumulates the more productive the User is.  

So even if a task is assigned to several people, this would in a way motive users with a lower score to complete the task

**Comment by xhk416x** [comment]
One solution to having multiple people assigned is having one "Owner" and several "assigned". This way the owner is overall responsible for the completion of the task, but the assigned people are specific team members (potentially other specialties) needed for that task.
